packages/react/src/components/Comments/Comment.tsx

```diff
@@ -235,10 +258,14 @@ export const Comment = ({
             </Components.Generic.Toolbar.Button>
           ))}
         {(canDeleteComment || canEditComment) && (
-          <Components.Generic.Menu.Root>
+          <Components.Generic.Menu.Root
+            position={"bottom-start"}
+            onOpenChange={(open) =>
```

Comment:
we now have this logic in two places (to make sure tooltip hides when clicked). Is this the cleanest way or can we find a way to make this the responsibility of one of the components themselves? or extract it in some other way that's reusable?